pom.xml

```diff
@@ -0,0 +1,52 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>io.github.utplsql</groupId>
+  <artifactId>java-api</artifactId>
+  <version>1.0</version>
+  <packaging>jar</packaging>
+
+  <name>utPLSQL-java-api</name>
+  <url>https://github.com/utPLSQL/utPLSQL-java-api</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.oracle.jdbc</groupId>
+      <artifactId>ojdbc7</artifactId>
```

Comment:
I would prefer to use Java 8 because of lambda expressions and try with resources. But if we want to support both drivers, we can't use those new features anyways.

We could start with the Java 7 and add support to the newest later. 

What do you think?